solutions/src/main/java/ru/sberbank/school/task08/KryoManager.java

```diff
@@ -33,7 +33,6 @@ public void initialize() {
 
     @Override
     public void saveGame(String filename, MapState<GameObject> gameState) throws SaveGameException {
-
         try (FileOutputStream fos = new FileOutputStream(filename);
              Output out = new Output(fos)) {
```

Comment:
&#39;try resource&#39; has incorrect indentation level 13, expected level should be 16.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a889404154>*